### PR TITLE
feature/nuc-119-mkapi-transfer-image-component

### PR DESCRIPTION
**Pull request by mkapi**
Transfer of Image Component

### CODE REVIEW

File: docs/components/image.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+
+## Props
+
+| Name      | Type                                      | Default | Web | Mobile | Description |
+| --------- | ----------------------------------------- | ------- | --- | ------ | ----------- |
+| src       | String                                    |         | ✔   | ✔      | Path of the image resource. Full URL for external image, relative URL starting with '~' for application asset. |
+| tintColor | String                                    |         | ✘   | ✔      | Tint color shown over image. |
+| stretch   | `none`, `aspectFill`, `aspectFit`, `fill` | `none`  | ✔   | ✔      | Image stretching strategy inside image container. |
+| loadMode  | `async`, `sync`                           | `async` | ✘   | ✔      | Image loading strategy. `async` fetches and loads image in background. |
```
Comment (ahmetceylan) [inline docs/components/image.md]: events are missing in documentation file

Comment (mkapi) [reply docs/components/image.md]: Added events in https://github.com/vuexp/vuexp/pull/35/commits/7ae4d00672d0863cb29e58b22763a10b18f89df2

**Comment by mkapi** [comment]
Removed Img.native.vue component from project with https://github.com/vuexp/vuexp/pull/35/commits/7393df43dbbb90cce75c5c2ca2e44fc53bc0e1c0

**Comment by mkapi** [comment]
> package-lock.json should not be pushed

reverted package-lock.json to the integration branch

**Pull request merged**